src/igvfd/schemas/mixins.json

```diff
@@ -484,6 +485,7 @@
                     "CRISPR FACS screen": "A CRISPR-based screen that employs Fluorescence-Activated Cell Sorting (FACS) to separate and analyze cell populations based on fluorescence markers.",
                     "CRISPR MACS screen": "A CRISPR-based screen that employs Magnetic-Activated Cell Sorting CRISPR screen (MACS) which uses magnetic beads conjugated to specific antibodies or markers to isolate and analyze cells based on surface protein expression.",
                     "CRISPR mCherry screen": "A CRISPR-based screen that uses mCherry fluorescent protein as a reporter to assess gene knockout, activation, or repression effects on cellular processes.",
+                    "CROP-seq": "CROP-seq captures CRISPR interference guide RNAs alongside cellular transcriptomes in single-cell RNA sequencing to link gene repression with transcriptional responses.",
```

Comment:
"alongside" I think does not really capture the integration of guide RNA expression directly into the single-cell transcriptome. regular Perturb-seq also captures guide RNAs "alongside" transcriptome. 

How about:
"A single-cell CRISPR screen that captures guide RNAs as part of the polyadenylated transcriptome, enabling direct linkage of perturbation and transcriptional response without separate barcoding."

Also can you move this down to be with the other single cell CRISPR assays, right below TAP-seq maybe?